include/executor.hpp

```diff
@@ -324,5 +336,10 @@ class Executor
     /* SRC and HEX words */
     std::string latestSrcAndHexwords;
 
+    /* To keep track if the execute request is from D-bus or not */
+    bool isExternallyTriggered = false;
+
+    types::FunctionExecutionResult executionStatus =
```

Comment:
why this is required in the class ?

Reply:
This is to keep track of executionStatus without changing the signature of existing apis - executeFunction() and displayExecutionStatus()